.github/workflows/e2e-suite.yml

```diff
@@ -82,8 +86,14 @@ jobs:
           pip install certifi -U && \
           pip install .[obj,dev]
 
+      - name: Set OpenAPI Spec URL
```

Comment:
Probably don't need this extra step if we input it directly. Refer to comment below